src/recon_buildblock/PoissonLogLikelihoodWithLinearModelForMeanAndListModeDataWithProjMatrixByBin.cxx

```diff
@@ -108,11 +107,11 @@ initialise_keymap()
 
   this->parser.add_key("num_events_to_use",&this->num_events_to_use);
   this->parser.add_key("max cache size", &cache_size);
-  this->parser.add_key("cache path", &cache_path);
   this->parser.add_key("recompute cache", &recompute_cache);
```

Comment:
Why is this here, but other cache cache variables one level up?

Reply:
I fixed this (and other similars)